src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -144,6 +194,17 @@ def help_text
           # rubocop:enable Metrics/MethodLength
         end
 
+        def separate_vgs_help_text
+          # TRANSLATORS: Help text for the partitioning scheme (use separate LVM Volume Groups)
+          _(
+            "<p>" \
+            "<b>Use Separate LVM Volume Groups for Some Special Paths</b> indicates " \
```

Comment:
:thinking: :thinking: 

Then, we should do the same with "Enable Disk Encryption" string, isn't it?

Reply:
Yes, that is better than repeating the same string in different places.

Reply:
I was working on it but, due to translations, I think that in this case it will be better to keep those repeated labels than "over complicating" the code.